topics/imaging/tutorials/imaging-introduction/workflows/Galaxy-Workflow-GTN_Tutorial__Introduction_to_Imaging.ga

```diff
@@ -1,786 +1,561 @@
-      "tool_version": "0.0.4",
-      "type": "tool",
-      "uuid": "5fb270a6-5608-4e06-8fa4-e6c2ec351cba",
-      "workflow_outputs": [
-        {
-          "label": null,
-          "output_name": "output",
-          "uuid": "c2547432-1a8e-425a-9207-5a1f1deb1f5f"
-        }
-      ]
-    },
-    "14": {
-      "annotation": "",
-      "content_id": "toolshed.g2.bx.psu.edu/repos/imgteam/bfconvert/ip_convertimage/0.4",
-      "errors": null,
-      "id": 14,
-      "input_connections": {
-        "input_file": {
-          "id": 11,
-          "output_name": "output"
-        }
-      },
-      "inputs": [
-        {
-          "description": "runtime parameter for tool Convert image",
-          "name": "input_file"
-        }
-      ],
-      "label": null,
-      "name": "Convert image",
-      "outputs": [
-        {
-          "name": "output",
-          "type": "tiff"
-        }
-      ],
-      "position": {
-        "left": 1546.61669921875,
-        "top": 1125.11669921875
-      },
-      "post_job_actions": {
-        "HideDatasetActionoutput": {
-          "action_arguments": {},
-          "action_type": "HideDatasetAction",
-          "output_name": "output"
-        }
-      },
-      "tool_id": "toolshed.g2.bx.psu.edu/repos/imgteam/bfconvert/ip_convertimage/0.4",
-      "tool_shed_repository": {
-        "changeset_revision": "b07f4839ca77",
-        "name": "bfconvert",
-        "owner": "imgteam",
-        "tool_shed": "toolshed.g2.bx.psu.edu"
-      },
-      "tool_state": "{\"out_format\": \"\\\"png\\\"\", \"__page__\": null, \"z_options\": \"{\\\"__current_case__\\\": 0, \\\"extract\\\": \\\"False\\\"}\", \"timepoint_options\": \"{\\\"__current_case__\\\": 0, \\\"extract\\\": \\\"False\\\"}\", \"channel_options\": \"{\\\"__current_case__\\\": 0, \\\"extract\\\": \\\"False\\\"}\", \"__rerun_remap_job_id__\": null, \"input_file\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"crop_options\": \"{\\\"__current_case__\\\": 0, \\\"extract\\\": \\\"False\\\"}\", \"series_options\": \"{\\\"__current_case__\\\": 0, \\\"extract\\\": \\\"False\\\"}\", \"range_options\": \"{\\\"__current_case__\\\": 0, \\\"extract\\\": \\\"False\\\"}\", \"tile_options\": \"{\\\"__current_case__\\\": 0, \\\"extract\\\": \\\"False\\\"}\"}",
-      "tool_version": "0.4",
-      "type": "tool",
-      "uuid": "5fb36cc6-797f-499e-be64-3e239eb4ab26",
-      "workflow_outputs": []
-    },
-    "15": {
-      "annotation": "",
-      "content_id": "toolshed.g2.bx.psu.edu/repos/imgteam/overlay_segmentation_mask/ip_overlay_segmentation/0.0.6",
-      "errors": null,
-      "id": 15,
-      "input_connections": {
-        "input_image": {
-          "id": 6,
-          "output_name": "output"
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "format-version": "0.1",
```

Comment:
Due to the linter complains:

1) Add a line with `"license": "MIT"` or some other OSI license here?

2) Add the a `Creaetor` block, like this or something similar?
```json
"creator":[
      {
         "class":"Person",
         "identifier":"https://orcid.org/xxxx-xxxx-xxxx-xxxx",
         "name":"xxx xxx",
         "url":"xxx"
      }
   ],
```

Reply:
Hi @kostrykin, these can most easily be filled in Galaxy itself, using the "Best Practices" tab which will give you a nice form for entering them correctly. 

Here's where you find the best practices:
![image](https://github.com/galaxyproject/training-material/assets/458683/0ad41b9f-8bd3-42dd-be4f-de3cd61d29aa)

Following through with the suggestions will let you fix all best-practice issues:
![image](https://github.com/galaxyproject/training-material/assets/458683/ca6b0529-349d-40bd-9c17-18a091aa5d13)

Or if you just want to fix the linting complaints and nothing else, this can be done from the edit tab:
![image](https://github.com/galaxyproject/training-material/assets/458683/6a936cfc-e322-4110-836e-bff186440180)